src/collections/_documentation/workflow/discover2/query-builder.md

```diff
@@ -0,0 +1,333 @@
+
+## Saved Query
+
+Although search queries are shareable, you can save them as Saved Queries. Saved Queries are visible to the entire organization. In other words, Saved Queries **are not** scoped to the user's account. Saved Queries have a yellow star icon on the Discover landing page. You can use either a pre-built query or another saved query to create a new Saved Query.
+
+### Creating a new Saved Query
+
+You may create a new Saved Query from one of the following ways:
+
+- From a pre-built query:
+    1. Click on "Save as..." on the top right
+    2. Enter the name of your Saved Query
+    3. Click "Save"
+- From a Saved Query:
+    1. Make changes to a Saved Query
+    2. Click on "Save as..." on the top right
+    3. Enter the name of your Saved Query
+    4. Click "Save"
+
+### Saving Changes
+
+Making changes to a Saved Query **will not** automatically be saved. An exception to this rule is renaming a Saved Query. Note that the state is still reflected in the URL so that you can share the URLs even though they're not saved. If you've made any unsaved changes to a Saved Query, you can click on the "Update Query" button on the top right corner.
+
+### Renaming a Saved Query
+
+1. Click on the Saved Query name with the pencil icon 
+2. Enter a new name
+3. Click enter or click outside to confirm changes
+4. The name is automatically saved
+
+### Deleting a Saved Query
+
+{% capture __alert_content -%}
+Deleting a Saved Query is irreversible.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Warning"
+    content=__alert_content
+    level="warning"
+%}
+
+From an existing Saved Query, click on the garbage bin icon on the top right-hand corner. After deleting a Saved Query, you'll be redirected back to the landing page.
+
+## Search Conditions
+
+You can use conditions to filter events from results by fields or tags.
+
+### Available Fields
+
+Events have a number of built in fields as well as custom tags.
+
+#### Standard fields
+
+<table class="table">
+{%- for category in page.standard_fields -%}
+  <tr>
+    <td>
+      <strong>{{ category[0] }}</strong>
+    </td>
+    <td class="content-flush-bottom">
+      {%- assign items = category[1]-%}
+      {%- include api/params.html params=items -%}
+    </td>
+  </tr>
+{%- endfor -%}
+</table>
+
+### Syntax
+
+The Query Builder syntax is identical to [Sentry's Search syntax]({%- link _documentation/workflow/search.md -%}). To add a search condition, click on the search input box above the graph, add a search condition, and press "Enter."
+
+### Searching on aggregated fields/tags
+
+You can add search conditions on aggregated fields/tags (for example, `count(id)`) in one of the following ways:
+
+- Exact match: `count(id):99`
+- Upper bounds: `count(id):<99` or `count(id):<=99`
+- Lower bounds: `count(id):>99` or `count(id):>=99`
+- Combination of upper and lower bounds: `count(id):>10 count(id):<20`
+
+## Results Table
```

Comment:
I think it would make more sense to me to put the ### Graph section in front of the ### Results Table section. That is how the page flows and the lower sections are more relevant to the results table than the graph.

Reply:
The Graph section is nested under the Results Table section for context because the Graph section references the Results Table. I think this makes more sense contextually. Let's keep it this way for now and iterate later if my contextual hypothesis proves wrong.